extensions/ql-vscode/src/query-results.ts

```diff
@@ -72,6 +71,10 @@ export class CompletedQuery implements QueryWithResults {
     }
   }
 
+  getResultsPath(selectedTable: string, useSorted = true): string {
+    return (useSorted && this.sortedResultsInfo.get(selectedTable)?.resultsPath)
```

Comment:
Any reason not to `useSorted ? this.sortedResultsInfo.get(selectedTable)?.resultsPath) : this.query.resultsPaths.resultsPath`? Am I missing something about the use of the short-circuiting boolean operators?

Reply:
I was just trying to be concise, but it's actually confusing to read. I think I'll make it longer and simpler.

This is not correct:

```js
useSorted ? this.sortedResultsInfo.get(selectedTable)?.resultsPath) : this.query.resultsPaths.resultsPath
```

because if `this.sortedResultsInfo.get(selectedTable)?.resultsPath)` is falsy then we need to fall back on `this.query.resultsPaths.resultsPath` hence the short-circuiting.

Reply:
Ah, yeah, less concise but more clear would be preferred, since in fact I did misread it by accident.